### ISSUE TITLE
Optimisation of Gallery: discriminate number of parameters etc

### ISSUE THREAD

**Issue by jgrooss** [Open-MSS/MSS#1560]
It would be nice to have the gallery display on the server somehow overseeable.
(1) In order to have a decent number of parameters be displayed it would be nice to have the opportunity to only display a subset of the available layers. Currently all available layers are plotted. In the PHILEAS setup, a choice of "all times, all levels" would result in the creation of over 16.000 top view plots. Therefore it would be nice to have an option like --layers a,b,c or better, have the list of displayable layers in a separate input file
(2) a standard version of a side view plot seems to be based on an arbitrary flight path diagonal through the model data domain, (lon_min, lat_min) to (lon_max, lat_max). This is likely not often needed.  It should be possible to switch off the creation and display of the side view plots on the gallery, or -if wanted- to base them on a given flight track (or even multiple given flight tracks as NS and EW cuts through the campaign location position).
(3) Same arguments hold for linear plots. These are unlikely necessary in the Gallery, at least, their creation and display should be able to be switched off. It is also not clear to me, on which flight path this plots are based.

**Comment by ReimarBauer** [comment]
Just a note: 

Once the GSOC project by @swsrkty is merged we have a variety of options for alternative galleries or alternatives to create plots or also on request. I assume a refactoring of the current will be done afterwards. 

For the interim we should maybe start with a "--layers a,b,c" option for the current gallery.

**Comment by ReimarBauer** [comment]
(1) is only depending on the users mswms_settings.py 
You can use an env var to seperate parts which should only be done when you run the gallery sub command.
(2) and (3) agree for a feature of disabling sideview and linearview images for real data. Other changes should be done after we have done a refactoring.
